.builders/build.py

```diff
@@ -113,13 +118,25 @@ def build_image():
 
             dependency_file = mount_dir / 'requirements.in'
             dependency_file.write_text('\n'.join(chain.from_iterable(read_dependencies().values())))
-            shutil.copy(HERE.parent / '.deps' / 'build_dependencies.txt', mount_dir)
+            shutil.copy(HERE / 'deps' / 'build_dependencies.txt', mount_dir)
             shutil.copytree(HERE / 'scripts', mount_dir / 'scripts')
             shutil.copytree(HERE / 'patches', mount_dir / 'patches')
 
+            # Create outputs on the host so they can be removed
+            wheels_dir = mount_dir / 'wheels'
+            wheels_dir.mkdir()
+            built_wheels_dir = wheels_dir / 'built'
+            built_wheels_dir.mkdir()
+            external_wheels_dir = wheels_dir / 'external'
+            external_wheels_dir.mkdir()
+            final_requirements = mount_dir / 'frozen.txt'
+            final_requirements.touch()
```

Comment:
Why do we need this exactly? I'm not sure I see why we would need to remove these anyway.

Reply:
The temporary directory gets removed is the issue